### PR TITLE
Add a jupyter notebook for visualizing ephys data

### PR DESCRIPTION
**Pull request by JaerongA**
- Added a jupyter notebook for visualizing ephys data.
- Added a jupytext paired script.
- Added `ephys_report` schema to the pipeline.
- Updated the pytest module with `ephys_report`
- Added a test for `build_electrode_layouts`

### CODE REVIEW

**Comment by JaerongA** [comment]
Thanks @CBroz1 for the comments. It all makes sense.. I'll see if I can address them.

**Comment by CBroz1** [comment]
I ran through the notebook. It looks good. It's a bit sparse, however, on the explanations. To the extent that we can, it reflects well on us if our notebooks demonstrate subject matter expertise. Even just a sentence here and there about why each graph is useful would go a long way.

I encountered the following error when running the main widget in a notebook via vscode
> `'DOMWidget' has no attribute '_ipython_display_'`
And [this post](https://github.com/polakowo/vectorbt/issues/488) suggested pinning some dependencies as follows
> `pip install "jupyterlab>=3" "ipywidgets>=7.6"; pip install plotly==5.10.0`
I don't know that his level of pinning is required, but we could think about adding a suggestion somewhere in the notebook to anticipate this error

**Comment by JaerongA** [comment]
hmm.. never saw that error. Maybe I'll add that to `requirements.txt`? I'll add some more docs where I can.

**Comment by CBroz1** [comment]
I think adding it to `requirements.txt` is overkill if you haven't seen it before. Maybe it's an M1 thing?

**Comment by JaerongA** [comment]
> I think adding it to `requirements.txt` is overkill if you haven't seen it before. Maybe it's an M1 thing?

Hmm not sure whether this would be M1 specific. Maybe @sidhulyalkar or @tdincer can give this a try?

**Review by CBroz1 — APPROVED** [review]
Thanks for your work on this @JaerongA. The additional notebook explanations are super helpful

**Comment by CBroz1** [comment]
Sorry - The last thing to do before merge is update the changelog. I think this can be scheduled for the next release

File: tests/test_populate.py [hunk 1]
```diff
@@ -329,6 +333,36 @@ def test_waveform_populate_npx3B_SpikeGLX(curations, pipeline, testdata_paths):
     assert waveforms.shape == (150, 64)
 
 
+@mark.jr
```
Comment (CBroz1) [inline tests/test_populate.py]: What does this decorator do in this context?

**Review by CBroz1 — APPROVED** [review]
Looks good. I'm just not familiar with the decorator here

**Pull request merged**